resources/definitions/anycubic_mega_zero2.def.json

```diff
@@ -0,0 +1,107 @@
+{
+    "version": 2,
+    "name": "Anycubic Mega Zero 2.0",
+    "inherits": "fdmprinter",
+    "metadata":
+    {
+        "visible": true,
+        "author": "su77ungr",
+        "manufacturer": "Anycubic",
+        "file_formats": "text/x-gcode",
+        "platform": "anycubic_mega_zero_platform.stl",
+        "has_materials": true,
+        "has_machine_quality": true,
+        "preferred_quality_type": "normal",
+        "preferred_material": "generic_pla",
+        "machine_extruder_trains":
+        {
+            "0": "anycubic_mega_zero2_extruder_0"
+        }
+    },
+
+    "overrides":
+    {
+        "machine_name":
+        {
+            "default_value": "Anycubic Mega Zero 2.0"
+        },
+        "machine_heated_bed":
+        {
+            "default_value": true
+        },
+        "machine_width":
+        {
+            "default_value": 220
+        },
+        "machine_depth":
+        {
+            "default_value": 220
+        },
+        "machine_height":
+        {
+            "default_value": 250
+        },
+        "gantry_height":
+        {
+            "value": 25
+        },
+        "machine_gcode_flavor":

```

Comment:
Don't commit this suggestion via the Github UI, because it will result in a ill-formed json.
 
 This is a bug in the the printer-linter which I will fix tomorrow morning

Reply:
Should I wait till next day; or fix it manually?

Reply:
tou can fix it manually if you agree with the suggestions in the first place

Reply:
remember these are all suggestions in the first place if you override the property  for a specific  reason you can just click resolve and leave it be.